src/blocks/Header/index.js

```diff
@@ -25,19 +35,15 @@ const Header = props => {
 
   const darkBgPages = ['/pricing', '/privacy-policy', '/terms-of-use']
   const hasDarkBgColor = darkBgPages.includes(router.asPath)
+  const banner = headerProps?.banner?.title
 
   return (
-    <Wrapper
-      style={{
-        backgroundColor: hasDarkBgColor
-          ? get('colors.valhalla.800')
-          : get('colors.valhalla'),
-      }}
-      data-cy="header"
-      {...props}
-    >
-      <DesktopHeader headerProps={headerProps} />
-      <MobileHeader headerProps={headerProps} />
+    <Wrapper data-cy="header" {...props}>
+      {banner && <Banner title={banner} />}
```

Comment:
Do you think it is worth adding the possibility to close the banner?

Reply:
We thought about it and decided not to give this possibility :)